.github/workflows/ci.yml

```diff
@@ -18,6 +18,9 @@ jobs:
           POSTGRES_USER: minimalkv_test
     env:
       SIMPLEKV_CI: 1
+      AWS_ACCESS_KEY_ID: minio
+      AWS_SECRET_ACCESS_KEY: miniostorage
+      AWS_DEFAULT_REGION: us-east-1
```

Comment:
shouldn't this be automatically read from [here](https://github.com/data-engineering-collective/minimalkv/blob/main/boto_credentials.ini)? why do we need to set them explicitly?